sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Release History
 
 ## 1.4.0b2 (Unreleased)
+- All `get_token` methods consistently require at least one scope argument,
+raising an error when none is passed. Although `get_token()` may sometimes
+have succeeded in prior versions, it couldn't do so consistently because its
+behavior was undefined, and dependened on the credential's type and internal
+state. ([#10243](https://github.com/Azure/azure-sdk-for-python/pull/10243))
```

Comment:
picky: GH is nice with us, but technically your URL contains "pull" even if it's not :)
https://github.com/Azure/azure-sdk-for-python/issues/10243